src/common/deleteme/collection.hpp

```diff
@@ -0,0 +1,8 @@
+//
```

Comment:
[Question]  This file is in a directory called "deleteme".  Is it supposed to be checked in?